npm2deb/__init__.py

```diff
@@ -353,6 +353,7 @@ def create_rules(self):
 
     def create_tests(self):
         utils.create_dir("debian/tests")
```

Comment:
`utils_create_dir` creates parent directories too.
you can remove line 355.

Reply:
Done, thanks